CMakeLists.txt

```diff
@@ -387,6 +392,13 @@ else()
     set(BuildJIT 1)
 endif()
 
+if(INTL_ICU_SH)
+    unset(INTL_ICU_SH CACHE)    # don't cache
+    unset(IntlICU CACHE)        # also clear it just in case
```

Comment:
I'm not that familiar with CMake. I welcome feedback for how to do this more effectively.